### DISCUSSION TITLE
Single board based on RP2040 in DIP-28 footprint?

### DISCUSSION THREAD

**Discussion by modeler** [frntc/SIDKick-pico#33] [Ideas]
Hi,

I understand you can buy the Pico's MCU as a stand alone part, SKU SC0914(13). I was wondering if it might be practical to produce a SIDKick PCB without needing the whole Pico board attached to the top?

This is both out of interest and for aesthetic reasons only. It would be good hear others thoughts on this.

Thanks!

**Comment by modeler** [comment]
Completely missed #27, sorry.

**Reply by frntc** [reply]
No problem. :-) Yes, such a version exists (actually two different ones), but I'll prefer careful testing before releasing anything. 

Edit: Let me add that a DIP-28/30 version with DAC, opamp, level shifters etc. is definitely not suitable for hand-soldering anymore. And the "simple-pcb" + Pico is unbeatable low cost.

**Reply by perhof** [reply]
I understand that the Pico option is cheaper and easier to build for the novice but it's too clunky to fit in a C128DCR (power supply is in the way) so I would much prefer a DIP-28 version.
If I have to choose between having the SIDkick Pico and having a power supply I will have to go with the power supply.
